tomo3d/protocols/protocol_denoise_tomogram.py

```diff
@@ -91,7 +96,9 @@ def _defineParams(self, form):
                            ' 10 iterations are assumed. This parameter is strongly related'
                            ' to the time step \n')
 
-        form.addParam('nIterBflow', IntParam, default=70,
+        form.addParam('nIterBflow', IntParam,
+                      default=70,
+                      validators=[GE(50),LE(250)],
```

Comment:
I do not see a need for validation in a range. This range is a recommendation but not mandatory